sonar-compat/src/main/java/org/sonar/java/JavaIssue.java

```diff
@@ -38,7 +38,7 @@ public JavaIssue(NewIssue newIssue) {
   public static JavaIssue create(SensorContext context, RuleKey ruleKey, @Nullable Double effortToFix) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SensorContext"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)